inst/rmarkdown/templates/arxiv/resources/template.tex

```diff
@@ -66,6 +66,13 @@
 \newcommand{\CSLIndent}[1]{\hspace{\cslhangindent}#1}
 $endif$
 
+% Pandoc header
+$for(header-includes)$
+$header-includes$
+$endfor$
+
+$preamble$
```

Comment:
`preamble` and `header-includes` are the same IMO. I don't think a template should have both - some of existing template have both but not sure why. 

Why do you think we need both ? 

I would rather only add `header-includes` (which #356 also suggest).